app/components/dashboard/run/left-panel/component.js

```diff
@@ -110,38 +109,30 @@ export default Component.extend(FullScreenMixin, {
         this.createTooltips();
     },
 
+  
+    /**
+     * Used to check for the ?auth=true query parameter and potentially
+     * open the publish modal
+     *
+     * @method didInsertElement
+    */
     didInsertElement() {
         scheduleOnce('afterRender', this, () => {
           // Check if we're coming from an ORCID redirect
           let queryParams = this.get('params')
           if (queryParams) {
             if (queryParams.auth === 'true') {
-              this.router.transitionTo({ queryParams: { auth: null }});
-              this.send('openPublishModal', this.model.taleId);
-            }
+              this.get('store').findRecord('tale', this.model.taleId, {
+                reload: true
+              })
+              .then(resp => {
+              this.send('openPublishModal', resp)
+            })
```

Comment:
Tale should already be available via the `this.model.tale` (and will be `this.model` after the 0.8 refactoring). Is there a reason to requery for it here?

Reply:
Since `this.model` is the instance, I'm pretty sure it only has the ID of the Tale. To make sure, I tried to log `this.model.tale` just before the request and I got `undefined`

Reply:
Ahh, my apologies... the Run controller grabs all of this data, but it appears to be missing a line [here](https://github.com/whole-tale/dashboard/blob/master/app/controllers/run.js#L14) to actually save the Tale model that we fetch.

In the refactored Run view this should be available via the model, so I can address tidying up on that branch.